fs/move.py

```diff
@@ -55,6 +59,31 @@ def move_file(
             resources (defaults to `False`).
 
     """
+    if src_fs.hassyspath(src_path) and dst_fs.hassyspath(dst_path):
```

Comment:
Ah! It's just occurred to me that this probably won't work if src_fs or dst_fs are strings? :stuck_out_tongue_winking_eye:  (i.e. FS-URLs)

Reply:
Good catch! On it.